README.md

```diff
@@ -110,7 +110,17 @@ yarn global add vercel
 vercel
 ```
 
-This builds the app then pushes it out to the cloud.
+This builds the app then pushes it out to the cloud. If you have not used Vercel before, you will be prompted to log-in using your email before deploying.
+
+## Troubleshooting
+
+### Installing Node.js on Windows
+
+It is highly recommended that you install the [Chocalety package manager](https://chocolatey.org/) before installing Node.js on Windows. After Cholatey is installed, make sure to check the 'Automatically install the necessary tools' checkbox when prompted during the Node.js installation. These tools are needed to build some ShinKGS dependencies.
```

Comment:
i see 3 different spellings of "chocolatey" in this paragraph.  I'm hoping at least one of them is right?

Reply:
Yikes! Okay, I've updated all spellings to match what is in the URL (which is correct only because I copied the link 😀)